### PR TITLE
[FS-237]: Cycle 28 Maintenance

### PR DESCRIPTION
**Pull request by ASafaeirad**
This PR cycles 28 maintenance.

### CODE REVIEW

**Comment by AmirabbasJ** [comment]
was there a reason you didn't name the components folder in the app's subdirectories `components` instead of `_components`?

**Comment by AmirabbasJ** [comment]
is `DestructiveButton` really a good name?
I mean `Destructive` implies more on damage, but I also don't have a suggestion really, someone suggested `Retract`.
I mean the word either falls under the `cancel` territory (it refers to the destruction, cancel and etc) or in the `undo` territory (it refers to the retaking, rethinking and etc)

File: libs/design/DestructiveButton/DestructiveButton.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import { DestructiveButton } from './DestructiveButton';
+
+export default {
+  component: DestructiveButton,
+} as Meta<typeof DestructiveButton>;
+
+type Story = StoryObj<typeof DestructiveButton>;
+
+export const Default: Story = {};
```
Comment (AmirabbasJ) [inline libs/design/DestructiveButton/DestructiveButton.stories.tsx:11]:
```suggestion
export const Default: Story = {
  args: {
    children: 'پاک کردن',
  },
};
```

**Comment by ASafaeirad** [comment]
> is `DestructiveButton` really a good name? I mean `Destructive` implies more on damage, but I also don't have a suggestion really, someone suggested `Retract`. I mean the word either falls under the `cancel` territory (it refers to the destruction, cancel and etc) or in the `undo` territory (it refers to the retaking, rethinking and etc) what about naming it `ErrorButton`

Nah, that's a common name for buttons in UI/UX

**Comment by ASafaeirad** [comment]
> was there a reason you didn't name the components folder in the app's subdirectories `components` instead of `_components`?

To be on top and different from sub-routs.

**Review by AmirabbasJ — APPROVED** [review]

**Pull request merged**